crates/sui-protocol-config/src/lib.rs

```diff
@@ -252,12 +252,22 @@ struct FeatureFlags {
     // If true, then the new algorithm for the leader election schedule will be used
     #[serde(skip_serializing_if = "is_false")]
     narwhal_new_leader_election_schedule: bool,
+
+    // A list of supported OIDC providers that can be used for zklogin.
+    #[serde(skip_serializing_if = "is_empty")]
+    zklogin_supported_providers: String,
```

Comment:
is there any reason this is not `Vec<String>`, or even better `BTreeSet<String>`?

Reply:
good point, revised